src/store/ducks/opensrp/hierarchies/index.tsx

```diff
@@ -355,7 +363,7 @@ export interface WithoutMetaFilters {
 /** prop filters to customize selector queries */
 export interface Filters {
   rootJurisdictionId: string /** specify which tree to act upon */;
-  planId?: string /** get selections that are associated with this planId */;
+  planId: string /** get selections that are associated with this planId */;
```

Comment:
IMO making planId required will require one to supply it even for selectors like `getTreeById` that does not use it,